source/features/show-github-action-next-time.tsx

```diff
@@ -0,0 +1,83 @@
+import './show-github-action-next-time.css';
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import {parseCron} from '@cheap-glitch/mi-cron';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getRepoURL, getRepoGQL, getCurrentBranch} from '../github-helpers';
+
+const getActionsSchedules = cache.function(async (): Promise<{[index: string]: string} | undefined> => {
+	const {repository: {object: {entries: actions}}} = await api.v4(
+		`repository(${getRepoGQL()}) {
+			object(expression: "${getCurrentBranch()}:.github/workflows") {
+				... on Tree {
```

Comment:
Does this support using “on Blob” directly? Perhaps “.github/workflows” will return both the folder and the objects in that folder

Reply:
Tested in the GraphQL explorer, couldn't get it to work without `...on Tree`. Then again, I'm a newbie at GraphQL.